### PR TITLE
Improve macOS/tvOS installation instructions

### PR DESCRIPTION
**Pull request by paulb777**
Improve docs after feedback at https://twitter.com/mxcl/status/977576224276058112

### CODE REVIEW

**Comment by wilhuff** [comment]
This seems like a reasonable band-aid, but I think we could overhaul this to talk more about installation  up top, including macOS and tvOS and then separately discuss development and development issues.

Something like 
  * Installation
    * Binary pod: iOS only (link to official docs)
    * Source pods: iOS, macOS, tvOS (what we've got now)
    * Carthage: (experimental)
  * Development (what we call Usage now)
  * Community (what's there today)

Basically: it doesn't much matter that source pods for non-iOS and Carthage are experimental and/or community supported, we should just put this all in one place so that when people come to figure out how to get it, the answer is one place.

**Comment by paulb777** [comment]
Thanks @wilhuff - I've reorganized as you've suggested and made the source pod instructions more prescriptive. PTAL
